exocom-server/bin/spec_ci

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
-set -e
+set -ex
 
 node_modules/o-tools-livescript/bin/build
 node_modules/o-tools/bin/lint
-node_modules/.bin/mocha --compilers ls:livescript "**/*spec.ls"
-bin/features --format pretty
+node_modules/.bin/mocha --compilers ls:livescript "**/*spec.ls" --reporter dot
+bin/features --format progress
```

Comment:
Let's change this to `node_modules/.bin/cucumber-js --tags ~@todo --format progress` to avoid messing up the toolchain for development.